controllers/orders.controller.js

```diff
@@ -139,5 +141,45 @@ export default {
         return res.status(500).send('Something went wrong')
       }
     }
+  },
+  changeStatus: async (req, res) => {
```

Comment:
if this controller changes status to specific status. Make it in the name
For example changeOrderStatusToApproval